tests/hopp/test_battery_ldes_dispatch.py

```diff
@@ -0,0 +1,175 @@
+    battery = Battery(site, config=config)
+    battery._dispatch = SimpleBatteryDispatch(model,
+                                              model.forecast_horizon,
+                                              battery._system_model,
+                                              battery._financial_model,
+                                              'battery',
+                                              HybridDispatchOptions())
+    
+    model.test_objective = pyomo.Objective(
+        rule=create_test_objective_rule,
+        sense=pyomo.maximize)
+
+    battery.dispatch.initialize_parameters()
+    battery.dispatch.update_time_series_parameters(0)
+    battery.dispatch.update_dispatch_initial_soc(battery.dispatch.minimum_soc)   # Set initial SOC to minimum
+    assert_units_consistent(model)
+    results = HybridDispatchBuilderSolver.glpk_solve_call(model)
+
+
+    with subtests.test("TerminationCondition"):
+        assert results.solver.termination_condition == TerminationCondition.optimal
+
+    with subtests.test("test_objective"):
+        assert pyomo.value(model.test_objective) == pytest.approx(expected_objective, 1e-5)
+
+    with subtests.test("charge_power"): # TODO this test does not make sense
+        sum_charge_power = sum(battery.dispatch.charge_power)
+        assert sum(battery.dispatch.charge_power) == pytest.approx(sum_charge_power, 1e-2)
+
+    with subtests.test("discharge_power"): # TODO this test does not make sense because it just tests the thing against itself
+        sum_discharge_power = sum(battery.dispatch.discharge_power)
+        assert sum(battery.dispatch.discharge_power) == pytest.approx(sum_discharge_power, 1e-2)
+
+    with subtests.test("round trip efficiency"): # TODO this test does not make sense because it just tests the thing against itself
+        assert (sum(battery.dispatch.charge_power) * battery.dispatch.round_trip_efficiency / 100.0
+                == pytest.approx(sum(battery.dispatch.discharge_power)))
+
+    battery.simulate_with_dispatch(48, 0)
+    for i in range(24):
+        with subtests.test(f"power[{i}]"):
+            dispatch_power = battery.dispatch.power[i] * 1e3
+            assert battery.outputs.P[i] == pytest.approx(dispatch_power, 1e-3 * abs(dispatch_power))
+
+    for i in range(45):
+        with subtests.test(f"battery.dispatch.soc[{i}]"):
+            dispatch_soc = battery.dispatch.soc[i]
+            assert battery.outputs.SOC[i] == pytest.approx(dispatch_soc, 1e-1 * abs(dispatch_soc))
+
+    with subtests.test("dispatch_lifecycles_per_day"):
+        assert battery.outputs.dispatch_lifecycles_per_day[0:2] == pytest.approx([0.75048, 1.50096], rel=1e-3)
+
+    with subtests.test("n_cycles[23]"):
+        assert battery.outputs.n_cycles[23] == 0
+    
+    # with subtests.test("n_cycles[47]"):
+    #     assert battery.outputs.n_cycles[47] == 1
+    # Not consistent with the way that cycles are updated in the LDES model
```

Comment:
I'd suggest removing this commented-out test or otherwise note why the cycles are updated differently

Reply:
@genevievestarke, can you address this?

Reply:
Done!